src/modules/logger/logged_topics.cpp

```diff
@@ -60,6 +60,7 @@ void LoggedTopics::add_default_topics()
 	add_topic("estimator_innovation_test_ratios", 200);
 	add_topic("estimator_innovation_variances", 200);
 	add_topic("estimator_innovations", 200);
+	add_topic("estimator_optical_flow_vel");
```

Comment:
```suggestion
	add_topic("estimator_optical_flow_vel", 200);
```

Limit default logging rate.